src/test/kotlin/com/igorwojda/string/caesarcipher/challenge.kt

```diff
@@ -4,7 +4,24 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun encodeCaesarCipher(str: String, shift: Int): String {
-    TODO("not implemented")
+
+    val aLetterDecimalCode = 'a'.toInt()
+    val zLetterDecimalCode = 'z'.toInt()
+
+    var encryptedText = ""
+
+    str.forEach {
+        val encryptedLetterDecimalCode = (it + (shift % 26)).toInt()
```

Comment:
Magic numbers should be extracted as file constants, e.g. private const val ALPHABET_LENGTH = 26